homeassistant/components/flume/sensor.py

```diff
@@ -40,7 +40,9 @@ async def async_setup_entry(
 
     flume_entity_list = []
     for device in flume_devices.device_list:
-        if device[KEY_DEVICE_TYPE] != FLUME_TYPE_SENSOR:
+        if (device[KEY_DEVICE_TYPE] != FLUME_TYPE_SENSOR) | (
```

Comment:
Please use a boolean operator, not a bitwise operator.

Reply:
You want to do an in not ==

I've done what you need in the dev branch of a pr. I'll post it here later today

Reply:
Isn't this what you're intending?
```suggestion
        if (device[KEY_DEVICE_TYPE] != FLUME_TYPE_SENSOR) or (
```

Reply:
<img width="391" alt="image" src="https://user-images.githubusercontent.com/6491743/190670213-65e3ac06-21e6-4925-bc6d-b6eeb5c9e90a.png">

Try this

Reply:
Please combine the `if` statements with `or` into one statement.

Reply:
And don't use `.keys()`. The default dict membership test is already the dict keys.